src/esp/sim/AbstractReplayRenderer.h

```diff
@@ -108,6 +113,17 @@ class AbstractReplayRenderer {
   // Retrieve the depth buffer as a CUDA device pointer. */
   virtual const void* getCudaDepthBufferDevicePointer();
 
+  std::shared_ptr<esp::gfx::DebugLineRender> getDebugLineRender(
+      unsigned envIndex);
+
+  esp::geo::Ray unproject(unsigned envIndex,
```

Comment:
It would be helpful to comment that this is a viewport point to world-space ray function.